_content/papers/devils-valley-to-omega-point_barrett.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Devil’s Valley to Omega Point: Reflections on the Emergence of a Theme from the Nō"
+authors: 
+  - "T.H. Barrett"
+editor: "Tadeusz Skorupski"
+external_url: "http://www.shin-ibs.edu/documents/bForum/v2/01Barrett.pdf"
+source: "https://www.shin-ibs.edu/research/archived-publications-and-research-projects/the-buddhist-forum/the-buddhist-forum-volume-ii/"
+drive_links:
+  - "https://drive.google.com/file/d/1wy-Q1ZEUY3h7_4FVxnETBfzlArM3P_Da/view?usp=share_link"
+formats: [pdf]
+course: nature
+tags:
+  - cosmology
+  - daoism
+  - culture
+  - comparative-studies
+  - chinese buddhism
+  - east asia
+year: 1992
+oclc: 61271660
+publisher: School of Oriental and African Studies
+address: "London"
+volume: 2
+from_book: "The Buddhist Forum VOLUME II Seminar Papers 1988–90"
```

Comment:
Oops! This one should be `booktitle: `... because `from_book` is only used when linking to a monograph that's also in the library. 😅  (Perhaps I should combine these two fields to avoid confusion? What do you think?)